config/environment.js

```diff
@@ -61,7 +62,8 @@ module.exports = function (environment) {
           baseUrl: "{{OAUTH_BASE_URL}}",
           scope: 'openid rrn vo profile abb_gelinktNotuleren',
           redirectUri: "{{OAUTH_REDIRECT_URL}}",
-          logoutUrl: "{{OAUTH_LOGOUT_URL}}"
+          logoutUrl: "{{OAUTH_LOGOUT_URL}}",
+          returnUrl: "{{OAUTH_SWITCH_URL}}"
```

Comment:
it could be this needs to be switched to switchUrl see https://github.com/lblod/ember-acmidm-login/issues/7